.github/workflows/stage-dataset.yml

```diff
@@ -10,23 +10,54 @@ on:
         type: boolean
         description: "Set to true to skip the file-info upload (save some processing time)"
```

Comment:
suggestion for clarity: "Skip the file-info upload to save some processing time. Only select if this isn't the first time uploading the dataset, and there are no changes to per cell file-info"